resources/views/pages-legacy/gameSearch.blade.php

```diff
@@ -1,143 +0,0 @@
-<?php
-
-use App\Models\System;
-
-authenticateFromCookie($user, $permissions, $userDetails);
-
-$maxCount = 50;
-
-$consoleList = System::gameSystems()->active()->get(['ID', 'Name'])->keyBy('ID')->map(fn ($system) => $system['Name']);
-$consoleList = $consoleList->sort();
-$consoleList->prepend('All Consoles', 0);
-
-$count = requestInputSanitized('c', $maxCount, 'integer');
-$offset = requestInputSanitized('o', 0, 'integer');
-$method = requestInputSanitized('p', 0, 'integer');
-$consoleID = requestInputSanitized('i', 0, 'integer');
-
-$gameData = getGameListSearch($offset, $count, $method, $consoleID);
```

Comment:
This function is no longer referenced anywhere and can be removed.

Reply:
Removed in latest.